src/pages/Edit/HashtagBox/index.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react';
+
+import * as S from './index.styles';
+
+import { CLIENT_MESSAGE } from '@/constants/message';
+import useCheckBlack from '@/hooks/useCheckBlank';
+
+const HashtagBox = ({ hashtagList, setHashtagList, showSnackbar }) => {
+  const [newHashtag, setNewHashtag] = useState('');
+  const { isBlank, isNull } = useCheckBlack();
+  const onChangeHashtag = e => {
+    setNewHashtag(e.target.value);
+  };
+  const addHashtag = ({ key }) => {
+    if (key === 'Enter') {
+      if (isBlank(newHashtag) || isNull(newHashtag)) {
+        showSnackbar(CLIENT_MESSAGE.ERROR.EMPTY_HASHTAG);
+        setNewHashtag('');
+        return;
+      }
+      if (hashtagList.length === 5) {
+        showSnackbar(CLIENT_MESSAGE.ERROR.FULL_HASHTAG_LIST);
+        setNewHashtag('');
+        return;
+      }
+      if (hashtagList.includes(newHashtag)) {
+        showSnackbar(CLIENT_MESSAGE.ERROR.REPUTATION_HASHTAG);
+        setNewHashtag('');
+        return;
+      }
+      setHashtagList(prev => [...prev, newHashtag]);
+      setNewHashtag('');
+    }
+  };
+  const deleteHashTag = name => {
+    setHashtagList(prev => prev.filter(hash => hash !== name));
+  };
```

Comment:
이미 상태로 받는 hashtagList가 있는데 굳이 prev를 받을 필요는 없겠죠..?

그리고 다른 곳은 hashtag로 사용해주셨는데 여긴 hashTag네요! T는 소문자로 바꿔봐요,,

```suggestion
  const deleteHashtag = name => {
    setHashtagList(hashtagList.filter(hashtag => hashtag !== name));
  };
```

Reply:
이미 상태로 받는 hashtagList가 있는데 굳이 prev를 받을 필요는 없겠죠..? -> 그래도 set을 쓸 때는 prev를 통해 수정하는 게 안정성 측면에서 더 낫지 않나요?? 

다른 곳은 hashtag로 사용해주셨는데 여긴 hashTag네요 -> 어머 이건 미스테이크

Reply:
만약 setter가 너무 느려서 해시태그를 삭제하는 속도가 추가되는 속도보다 빠르면 최신화가 필요할 수 있지만, 저희의 경우에는 이미 추가되어 보여지는 해시태그만 클릭하여 삭제할 수 있으니 여기에서는 불필요한 부분이라고 생각해요!

Reply:
알겠습니다!!